ui/components/multichain/app-header/app-header-unlocked-content.tsx

```diff
@@ -101,10 +101,16 @@ export const AppHeaderUnlockedContent = ({
 
   // Passing non-evm address to checksum function will throw an error
   const normalizedCurrentAddress = normalizeSafeAddress(currentAddress);
-  const [copied, handleCopy] = useCopyToClipboard(MINUTE) as [
-    boolean,
-    (text: string) => void,
-  ];
+  const [copied, handleCopy, resetCopyState] = useCopyToClipboard(MINUTE, {
+    expireClipboard: false,
+  });
```

Comment:
Added this logic as it didn't really make sense that we "expire"/wipe the clipboard after a minute for public addresses. Other flows that rely on this hook still function as they did before (the SRP copy/paste still expires)